src/Forms/ListboxField.php

```diff
@@ -158,4 +167,109 @@ public function getDisabledItems()
     {
         return $this->disabledItems;
     }
+
+     /**
+     * Provide ListboxField data to the JSON schema for the frontend component
+     *
+     * @return array
+     */
+    public function getSchemaDataDefaults()
+    {
+        $options = $this->getOptions(true);
```

Comment:
Instead of passing an argument to `getOptions()` or (as per my suggestion in that method's changes) making a new method - you could add a filter here `$options->filter('Selected', true)`